### ISSUE TITLE
Color contrast issue with the currently playing song

### ISSUE THREAD

**Issue by garrett** [xou816/spot#156]
**Describe the bug**

It's almost impossible to read the currently playing song in playlists, due to the colors chosen.

**To Reproduce**
1. Visit a playlist
2. Play a song
3. See error

**Expected behavior**

The color choice should be at least AA compliant with accessibility standards. It should be easy for everyone to read.

**Screenshots**

![Screenshot from 2021-03-16 14-16-29](https://user-images.githubusercontent.com/10246/111317907-db5ba500-8664-11eb-90fd-ef9019454c21.png)

Here's a crop of the problem area:

![Screenshot from 2021-03-16 14-16-29 crop playlist](https://user-images.githubusercontent.com/10246/111319354-3215ae80-8666-11eb-98c7-f3aaa56b979b.png)

**General information:**
 - Distribution: Fedora 34 beta, Silverblue
 - Desktop: GNOME
 - Installation method: Flatpak, Flathub
 - Version: 0.1.10
 - Device used: Desktop

**Additional context**

If the same color would be applied to the background, it'd be a lot more readable. Perhaps something like this quick mockup was the intent?

![Screenshot from 2021-03-16 14-16-29 mockup](https://user-images.githubusercontent.com/10246/111319138-f975d500-8665-11eb-8022-bd1eee0138f2.png)

**Comment by xou816** [comment]
Thanks for the report! Yup, that's less than ideal :/ The blue background as per your suggestion is already in use for the selection mode (which you can enter by clicking the check mark button)

I thought about replacing the track number by something else, such as a "play" icon, but that might be too subtle when browsing a long list?

**Comment by garrett** [comment]
Oh, right. Selection. Good point!

It's probably worth looking at what the official Spotify client, GNOME Music, and Lollypop all do.

**Comment by xou816** [comment]
![Capture d’écran du 2021-03-19 10-17-28](https://user-images.githubusercontent.com/6277494/111758106-fd873a00-8893-11eb-8977-3603ea539735.png)
^ doing this for now! Not 100% satisfied as it's not visually striking enough to see it when scrolling long content, but that's better still

**Comment by xou816** [comment]
Fixed, will be in next version!
